en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-kubernetes-helm-artifacts.md

```diff
@@ -0,0 +1,135 @@
+# Deploying Choreo Connect as a Standalone Gateway on Kubernetes - Helm Artifacts
+
+Let's deploy an API, using WSO2 API Controller (apictl), on Choreo Connect, which running on Kubernetes as a Standalone Gateway.
+
+## Before you begin
+    
+-   Be sure you have an active [WSO2 Subscription](https://wso2.com/subscription). If you don't already have a subscription, sign up for a [WSO2 Free Trial Subscription](https://wso2.com/free-trial-subscription).
```

Comment:
This gives the idea that ppl need to have a valid sub for using helm charts. Can we improve this as in https://github.com/wso2/kubernetes-microgateway/blob/1.0.x/helm/choreo-connect/README.md#prerequisites